lib/bootloader_setup.pm

```diff
@@ -380,7 +380,89 @@ sub uefi_bootmenu_params {
     # assume bios+grub+anim already waited in start.sh
     # in grub2 it's tricky to set the screen resolution
     send_key 'e';
-    assert_screen("grub2-enter-edit-mode", 30) if is_jeos;
+    assert_screen("grub2-enter-edit-mode", 30) if (is_jeos || get_var('USB_BOOT'));
```

Comment:
There are so man uses of `get_var('USB_BOOT')`. I think we should have function `is_usb_boot` to be compatible with existing style.

Reply:
fixed